book/from_flat_php_to_symfony2.rst

```diff
@@ -5,10 +8,11 @@ Symfony versus Flat PHP
 
 **Why is Symfony better than just opening up a file and writing flat PHP?**
 
-If you've never used a PHP framework, aren't familiar with the MVC philosophy,
-or just wonder what all the *hype* is around Symfony, this chapter is for
-you. Instead of *telling* you that Symfony allows you to develop faster and
-better software than with flat PHP, you'll see for yourself.
+If you've never used a PHP framework, aren't familiar with the
+model-view-controller `MVC`_ philosophy, or just wonder what all the *hype*
```

Comment:
I would change this to: "Model-View-Controller (MVC) philosophy" also, we don't use a serial comma (the one before "or") in the docs. Although that isn't your fault, it would be nice to include it in this PR